project/profiles/capacity4more/modules/c4m/search/c4m_search/c4m_search.features.inc

```diff
@@ -380,6 +380,115 @@ function c4m_search_default_search_api_index() {
     "entity_type" : "search_api_index",
     "rdf_mapping" : []
   }');
+  $items['search_comments'] = entity_import('search_api_index', '{
```

Comment:
Could you please rename it to c4m_search_comments to be consistent with the other indexes?